web_full_stack/js/.node-version

```diff
@@ -1 +1 @@
-10.13.0
+16.16.0
```

Comment:
@hspearman I tried using the latest version but ran into other issues. This is the LTS, though.